### PR TITLE
A sample proposal to help potential speakers

### PR DESCRIPTION
**Pull request by mjang**

### CODE REVIEW

File: docs/conf/example-proposal.rst
Comment (Bradamant3) [inline docs/conf/example-proposal.rst]: s/than/rather than (or some edit or other to larger sentence that I couldn't quite get to ...)

**Comment by Bradamant3** [comment]
Nice nice nice and thank you Mike! Not gonna approve bc more folks should take a look first ...

File: docs/conf/example-proposal.rst
Comment (nbkaplan) [inline docs/conf/example-proposal.rst]: Change "hints" to "details"?

File: docs/conf/example-proposal.rst
Comment (nbkaplan) [inline docs/conf/example-proposal.rst]: "that include information about using specific tools."

Comment (nbkaplan) [reply docs/conf/example-proposal.rst]: Or "information specific to certain tools" ?

Comment (plaindocs) [reply docs/conf/example-proposal.rst]: Or "talks that focus on a specific software tool" ?

**Comment by plaindocs** [comment]
This is a great start @mjang thanks! 

My first gut reaction was that this is mostly restating what we say at great length in the CFP page, whereas I was hoping for something more specific, but the more I dig into it the more this approach makes sense.

I think we can just include this in the CFP page as something like `Proposal Structure`, what do you think? 

Probably worth being super explicit about which fields are public (Abstract) and which are meant for the CFP panel (all the others) :-D.

**Comment by mjang** [comment]
Appreciated, Sam, Jennifer, and Neal! Actually, Sam, when you made the suggestion, I thought, gosh, that's what I need as someone who writes proposals, a template with ideas in one screen that I can compare with my work in progress. 

I did try to incorporate existing guidance when I saw it. 

I'm good with your thought w/r/t "Proposal Structure" in the CfP page, but I'm just a little worried about length. (Is there a CfP template page? All I see are cfp.rst files for various conferences. I can update this PR to modify the cfp.rst page for Prague 2020, if you like; maybe as an `include` on that page.)

And yes, if you asked me "which fields get shown to the public?", I'd have to think about it.

File: docs/conf/example-proposal.rst
Comment (mjang) [reply docs/conf/example-proposal.rst]: I took that from some other text on the site (which should also be modified).

Comment (mjang) [reply docs/conf/example-proposal.rst]: I've also included `rather` in the prague/2020/cfp.rst wording, from where I took this. While it should be included in other past cfp.rst files, that's in the past. (Do we have an actual cfp.rst template, or is that a different issue?)

**Comment by plaindocs** [comment]
Yeah, we probably need to move the content of the CFP page to an include like the rest of them.

File: docs/conf/example-proposal.rst
Comment (plaindocs) [reply docs/conf/example-proposal.rst]: We leave past years as they are, and just copy/update the current year.

File: docs/conf/example-proposal.rst
Comment (plaindocs) [inline docs/conf/example-proposal.rst]:
```suggestion
Include a brief story, typically two to four paragraphs, that describes your personal work experience with the topic. Make sure the abstract appeals to our audience of documentarians. We're suspicious of proposals that look like they belong at other conferences.
```

**Review by plaindocs — APPROVED** [review]
Any further comments before we merge this in?

**Comment by mjang** [comment]
> Any further comments before we merge this in?

I'm good with the way you've set things up, @plaindocs . Thanks!

File: docs/conf/portland/2020/example-proposal.rst
Comment (eirinikos) [inline docs/conf/portland/2020/example-proposal.rst]: Found a typo! Please change `pratical` to `practical` :)

File: docs/conf/portland/2020/example-proposal.rst
Comment (eirinikos) [inline docs/conf/portland/2020/example-proposal.rst]: Found two typos!
* please change `audence` to `audience`
* please change `confernces` to `conferences`

File: docs/conf/portland/2020/example-proposal.rst
Comment (eirinikos) [inline docs/conf/portland/2020/example-proposal.rst]: Suggestion:

> Our audience creates documentation primarily for software. Given the variety of tools used for software documentation, we are wary about accepting talks that focus on a specific software tool **or set of tools. Talks that discuss tools should also discuss the wider context, applications, and implications of implementing the tools.**

File: docs/conf/portland/2020/example-proposal.rst
Comment (eirinikos) [inline docs/conf/portland/2020/example-proposal.rst]: Suggestion:

> This is the content that Write the Docs would post **to the conference website. In order to evaluate the quality of your proposal, the proposal committee will also take into account all of the info that you provide beyond the abstract.**

File: docs/conf/portland/2020/example-proposal.rst
Comment (eirinikos) [inline docs/conf/portland/2020/example-proposal.rst:38]: What do you think of adding an explicit note to the end of this section? e.g.,
`Only the proposal committee will see the info that you include here.`

File: docs/conf/portland/2020/example-proposal.rst
Comment (eirinikos) [inline docs/conf/portland/2020/example-proposal.rst:60]: What do you think of adding an explicit note to the end of this section? e.g.,
`Only the proposal committee will see the info that you include here.`

**Review by eirinikos — CHANGES_REQUESTED** [review]

**Comment by eirinikos** [comment]
@mjang I went ahead and fixed some typos in both copies of `example-proposal.rst` :+1: 

As for my suggestions - I left these comments only on the Portland copy of `example-proposal.rst` :)

File: docs/conf/portland/2020/example-proposal.rst
Comment (mjang) [reply docs/conf/portland/2020/example-proposal.rst]: I think this would invite more tooling talks, so I don't think this is wise. However,

@plaindocs If you think a substantial number of tooling talks are inevitable, @eirinikos 's option makes sense.

Comment (plaindocs) [reply docs/conf/portland/2020/example-proposal.rst]: Hmmm, I think a little leeway is a good thing. We could always roll back if it doesn't work out. :-D

File: docs/conf/portland/2020/example-proposal.rst
Comment (plaindocs) [inline docs/conf/portland/2020/example-proposal.rst]:
```suggestion
This is the content that Write the Docs would post on the conference schedule. To evaluate the quality of your proposal, the proposal committee also takes into account all of the other information that you provide.
```

File: docs/conf/portland/2020/example-proposal.rst
Comment (plaindocs) [inline docs/conf/portland/2020/example-proposal.rst]:
```suggestion
Our audience creates documentation primarily for software. Given the variety of tools used for software documentation, we rarely accept talks that focus on a specific software tool or set of tools. Talks that discuss tools should also discuss the wider context, applications, and implications of implementing the tools.
```

File: docs/conf/portland/2020/example-proposal.rst
Comment (plaindocs) [inline docs/conf/portland/2020/example-proposal.rst:39]:
```suggestion
 Only the proposal committee will see the information that you include here.
 
```

File: docs/conf/portland/2020/example-proposal.rst
Comment (plaindocs) [inline docs/conf/portland/2020/example-proposal.rst:60]:
```suggestion
* Teachers

 Only the proposal committee will see the information that you include here.
```

**Comment by plaindocs** [comment]
Wow so this has been here a while. Sorry folks. 
I'm gonna merge this in unless any objections are raised.

**Comment by eirinikos** [comment]
Thanks for the follow-up! I went ahead and applied the changes to the Prague `rST` file to stay consistent with the Portland file. :pray:

**Review by eirinikos — APPROVED** [review]

**Pull request merged**
